completion/services.py

```diff
@@ -92,7 +130,8 @@ def vertical_is_complete(self, item):
 
         # this is temporary local logic and will be removed when the whole course tree is included in completion
         child_locations = [
-            child.scope_ids.usage_id for child in item.get_children() if child.scope_ids.block_type != 'discussion'
+            child.scope_ids.usage_id for child in self.get_user_children(item)
+            if child.scope_ids.block_type != 'discussion'
```

Comment:
It seems potentially appropriate to move the `discussion` check inside `get_user_children`? Seems similar to the special handling for libraries.

Reply:
Yeah, I can do that. It was a remnant from the old version.